### PR TITLE
Display all/active/inactive and filter the datatable

### PR DESCRIPTION
**Pull request by bliuredhat**
Fix the bug from QE

JIRA:PDC-1589

### CODE REVIEW

**Comment by simozhan** [comment]
LGTM

File: pdc/apps/release/views.py [hunk 1]
```diff
@@ -562,7 +570,11 @@ class ProductVersionListView(SearchView):
     allow_empty = True
     template_name = "product_version_list.html"
     context_object_name = "product_version_list"
-    paginate_by = settings.ITEMS_PER_PAGE
+
+    def get_context_data(self, **kwargs):
```
Comment (ycheng-aa) [inline pdc/apps/release/views.py]: The above `get_context_data` almost same. Could implement it in a mixin and let each view inherit the mixin? Or some other way to eliminate duplicate code.

**Comment by ycheng-aa** [comment]
Generally looks to me. It's better to eliminate duplicate code.

**Comment by ycheng-aa** [comment]
:+1:

**Pull request merged**